src/graphql/schema.gql

```diff
@@ -208,6 +208,7 @@ type Space {
   voting: SpaceVoting
   categories: [String]
   validation: Strategy
+  followerCount: Int
```

Comment:
I think it should be with a s, like this "followersCount"
```suggestion
  followersCount: Int
```

Reply:
Looks like what Twitter API is using https://developer.twitter.com/en/docs/twitter-api/v1/data-dictionary/object-model/user

![image](https://user-images.githubusercontent.com/16245250/150130060-b325acf3-36c9-417c-b40a-4b83931e78bf.png)